go.mod

```diff
@@ -1,16 +1,15 @@
 module github.com/aquasecurity/tracee
 
-go 1.21
-
-toolchain go1.21.5
```

Comment:
Didn't remove it, just ran go mod tidy.
I will check it.

Reply:
@geyslan is the `toolchain` still a requirement?
It is automatically removed by my IDE (Goland).

Reply:
Actually the [PR](https://github.com/aquasecurity/tracee/pull/3999) is still open, I didn't have time to resolve its conflicts yet - mostly because it has to be coordinated with an internal tests bump (types).

> @geyslan is the toolchain still a requirement?

I think that it is a sane requirement since we (contributors) will always have the same go version for tests/debugging.

> It is automatically removed by my IDE (Goland).

Check if the cli `go mod tidy` removes it. In my case it doesn't.

![image](https://github.com/aquasecurity/tracee/assets/3372117/7e088f7a-9dd0-4211-b6a7-d56afed94afc)

I suppose that the maximum that go will enforce you is to download (automatically) the toolchain for that specific go.mod project. When you leave tracee folder, go version will be system's.

Reply:
Have you checked it in this PR branch?
Because it is reproduced in my machine with go mod tidy.
My go version is `go version go1.21.6 linux/amd64`

Reply:
![image](https://github.com/aquasecurity/tracee/assets/87267148/068f9cf9-d83d-4b21-bfec-9f2bc2ef2b7e)

Reply:
So go mod tidy reinserted the toolchain, right?

What is the output of `go version` in other folder (without a go.mod)?

Reply:
`go version go1.21.6 linux/amd64`